lib/rucio/daemons/badreplicas/necromancer.py

```diff
@@ -145,8 +148,11 @@ def necromancer(thread=0, bulk=5, once=False, sleep_time=60):
                         try:
                             update_rules_for_lost_replica(scope=scope, name=name, rse_id=rse_id, nowait=True)
                             monitor.record_counter(name='necromancer.badfiles.lostfile')
-                        except DatabaseException as error:
-                            logger(logging.WARNING, str(error))
+                        except (DatabaseException, DatabaseError) as error:
+                            if re.match('.*ORA-00054.*', error.args[0]) or re.match('.*ORA-00060.*', error.args[0]) or 'ERROR 1205 (HY000)' in error.args[0]:
```

Comment:
Wondering if we should do something like
https://github.com/rucio/rucio/blob/47e93cfbe5887071c70de4ba815c1bbdddfac2ce/lib/rucio/daemons/judge/cleaner.py#L100
in this case too.